### PR TITLE
Specs work contd.

### PR DESCRIPTION
**Pull request by cwgoes**
- [x] Fix LaTeX rendering

### CODE REVIEW

File: documentation/specs/src/base-ledger/multisignature.md [hunk 1]
```diff
@@ -68,12 +68,7 @@ Finally, the tx performs the following writes to storage:
 - The threshold
 - The list of public keys of the signers
 
-`Internal` addresses may want a multi-signature scheme on top of their validation process as well. Among the internal ones, `PGF` will require multisignature for its council (see the [relative](../economics/public-goods-funding.md) spec). The storage data necessary for the correct working of the multisig for an internal address are written in the genesis file: these keys can be later modified through governance.
-
-`Implicit` addresses are not generated by a transaction and, therefore, are not suitable for a multisignature scheme since there would be no way to properly construct them. More specifically, an implicit address doesn't allow for:
-
-- A custom, modifiable VP
-- An initial transaction to be used as an initializer for the relevant data
+Multisignature accounts can also be initialised at genesis time - in this case, the requisite parameters are kept in the genesis file and written to storage during initialisation.
```
Comment (cwgoes) [inline documentation/specs/src/base-ledger/multisignature.md:71]: @grarco I don't think we should need any special logic for PGF multisignatures now, right?

Comment (Fraccaman) [reply documentation/specs/src/base-ledger/multisignature.md:71]: no special logic needed

Comment (Fraccaman) [reply documentation/specs/src/base-ledger/multisignature.md:71]: but im not sure we can initialize a multisignature at genesis

Comment (cwgoes) [reply documentation/specs/src/base-ledger/multisignature.md:71]: why not?

**Comment by adrianbrink** [comment]
@cwgoes Can this be merged like this? I'd either like to merge it right now or close it unless you plan to add more things in the coming days.

**Comment by cwgoes** [comment]
> @cwgoes Can this be merged like this? I'd either like to merge it right now or close it unless you plan to add more things in the coming days.

I've updated the branch; as soon as the tests pass it can be merged.

**Pull request merged**